Source/Core/Core/HW/VideoInterface.cpp

```diff
@@ -505,39 +492,40 @@ static void BeginField(FieldType field)
 	// What should actually happen is that we should pass on the correct width,
 	// stride, and height to the video backend, and it should deinterlace the
 	// output when appropriate.
-	u32 fbStride = m_PictureConfiguration.STD * (field == FIELD_PROGRESSIVE ? 16 : 8);
+
+	// Below is a hack, but it adjusts interlaced fields to halve the stride and double the
```

Comment:
It would be more helpful for the reader to first explicitly describe what should actually be done and then what we do instead, and why that is a hack and why it works fine regardless of being a hack (.. if it does work fine regardless of that). Similarly, it would be nice to elaborate on what "needs to be smarter" means specifically.